gef-ui/server/utils/mocks/mock_requests.js

```diff
@@ -4,7 +4,7 @@ const { MAKER, CHECKER } = require('../../constants/roles');
 const chance = new Chance();
 
 const MOCK_REQUEST = {
-  username: 'BANK1_MAKER1',
+  username: 'maker1@ukexportfinance.gov.uk',
```

Comment:
We don't call from `users.js` in data loader here as these mocks (unit mocks/api mocks) should not/do not interact with the db.